recipe/meta.yaml

```diff
@@ -49,7 +49,7 @@ test:
 about:
   home: http://github.com/pymc-devs/pymc3
   license: Apache-2.0
```

Comment:
`pymc3` sdist packages: `RELEASE-NOTES.md`, `MANIFEST.in`, `README.rst`, `GOVERNANCE.md`, `CONTRIBUTING.md`, and `CODE_OF_CONDUCT.md` but it does not package the license file!!!

Reply:
cc: @twiecki I think we need to package the license in the release. Can you cut a patch release that includes the license file?

Reply:
There is no rush. This can be done in a next release. I added the license manually here for now.

PS-rant: PyPI encourages bad practices for not requiring people to ship the licenses with the packages but license like Apache-2.0 require the file to be there. Because PyPI packages are uploaded by the authors no one will sue themselves but, conda-forge is a community, so we need to do what the license requires us to do.